src/angular-nps.npsform.js

```diff
@@ -1,14 +1,52 @@
-(function() {
-    'use strict';
 
+(function () {
     angular.module('angular-nps')
-        .directive('npsForm', NpsForm);
+        .directive('npsForm', npsDirective)
```

Comment:
Check with the Angular coding standards I think the implementing functions are supposed to be proper case